app/views/wizard/index.html.erb

```diff
@@ -16,5 +16,12 @@
   <body class='wizard'>
     <div id='wizard-main'></div>
     <%= preload_script 'wizard-start' %>
+
+    <div id="svg-sprites" style="display:none;">
+      <div class="fontawesome">
+        <%= raw SvgSprite.bundle %>
```

Comment:
hmmm so the wizard bundles ? why is is not loading via ajax?

Reply:
I opted to use the bundle on the wizard to keep a smaller code footprint. To get the same AJAX-loading into the wizard would require exposing the SvgSprite path to the wizard JS, including the the sprite loader and adding an initializer to the wizard, the equivalent of `svg-sprite-fontawesome.js.es6`.